src/main/java/frc/trigon/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,132 @@
+
+import java.util.function.DoubleSupplier;
+
+public class IntakeConstants {
+    private static final int
+            MASTER_MOTOR_ID = 16,
+            FOLLOWER_MOTOR_ID = 17,
+            DISTANCE_SENSOR_CHANNEL = 0;
+    private static final String
+            MASTER_MOTOR_NAME = "MasterIntakeMotor",
+            FOLLOWER_MOTOR_NAME = "FollowerIntakeMotor",
+            DISTANCE_SENSOR_NAME = "IntakeDistanceSensor";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(
+            MASTER_MOTOR_ID,
+            MASTER_MOTOR_NAME,
+            RobotConstants.CANIVORE_NAME
+    ),
+            FOLLOWER_MOTOR = new TalonFXMotor(
+                    FOLLOWER_MOTOR_ID,
+                    FOLLOWER_MOTOR_NAME,
+                    RobotConstants.CANIVORE_NAME
+            );
+    static final SimpleSensor DISTANCE_SENSOR = SimpleSensor.createDutyCycleSensor(DISTANCE_SENSOR_CHANNEL, DISTANCE_SENSOR_NAME);
+
+    private static final InvertedValue
+            MASTER_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive,
+            FOLLOWER_MOTOR_INVERTED_VALUE = InvertedValue.CounterClockwise_Positive;
+    private static final boolean FOLLOWER_MOTOR_OPPOSITE_DIRECTION = false;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Coast;
+    private static final double GEAR_RATIO = 1.5;
+    private static final double
+            DISTANCE_SENSOR_SCALING_SLOPE = 0.0004,
+            DISTANCE_SENSOR_SCALING_INTERCEPT_POINT = 400;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int MOTOR_AMOUNT = 2;
+    private static final DCMotor GEARBOX = DCMotor.getFalcon500Foc(MOTOR_AMOUNT);
+    private static final double MOMENT_OF_INERTIA = 0.003;
+    private static final SimpleMotorSimulation SIMULATION = new SimpleMotorSimulation(
+            GEARBOX,
+            GEAR_RATIO,
+            MOMENT_OF_INERTIA
+    );
+    private static final double NOTE_DISTANCE_THRESHOLD_CENTIMETERS = 5;
+    private static final DoubleSupplier DISTANCE_SENSOR_SIMULATION_VALUE_SUPPLIER = () -> SimulationObjectDetectionCameraIO.HAS_OBJECTS ? NOTE_DISTANCE_THRESHOLD_CENTIMETERS - 1 : NOTE_DISTANCE_THRESHOLD_CENTIMETERS + 1;
+
+    private static final double MAX_DISPLAYABLE_VELOCITY = 12;
+    static final SpeedMechanism2d MECHANISM = new SpeedMechanism2d(
+            "IntakeMechanism", MAX_DISPLAYABLE_VELOCITY
+    );
+
+    static final double RUMBLE_DURATION_SECONDS = 0.6;
+    static final double RUMBLE_POWER = 1;
+    static final double NOTE_DETECTION_CONFIRMATION_DELAY_SECONDS = 0.6;
+    static final BooleanEvent BOOLEAN_EVENT = new BooleanEvent(
+            CommandScheduler.getInstance().getActiveButtonLoop(),
+            () -> DISTANCE_SENSOR.getScaledValue() < NOTE_DISTANCE_THRESHOLD_CENTIMETERS
+    ).debounce(NOTE_DETECTION_CONFIRMATION_DELAY_SECONDS);
+    static final double COLLECTION_CONFERMATION_SECONDS = 1;
+
```

Comment:
This is not for confirming collection, this is a wait to make sure the note stops (using brake) for enough time.